pagopa/pn-frontend-e2e
# Annullamento-notifica-e-controllo-download-pf

## Short description

Implemented one test to check the file download of one cancelled notification PF

## List of changes proposed in this pull request

- New steps to check the file download in one cancelled notification for PF
- Check on "allegati" PN-10255
- Check on AAR PN-10256
- Check on attestazioni opponibili a terzi PN-10257
- Check on "ricevute PEC" PN-10261
- Download multi F24 in a PF notification PN-10397

## How to test

Simply run `mvn clean test -D"cucumber.filter.tags=@checkDownloadFileNotificaAnnullata"` and verify if the tests works fine.
